traffic_portal/test/integration/PageObjects/CacheGroup.po.ts

```diff
@@ -158,6 +156,26 @@ export class CacheGroupPage extends BasePage {
         await snp.NavigateToCacheGroupsPage();
         return result;
     }
+    public async CheckCSV(name: string): Promise<boolean> {
+        return element(by.cssContainingText("span", name)).isPresent();
+    }
+    public async ToggleTableColumn(name: string): Promise<boolean> {
+        let result = false;
+        await this.btnTableColumn.click();
+        //if it's already on, turn it off and return false
+        if (await browser.isElementPresent(element(by.xpath("//th[text()='" + name + "']"))) === true){
```

Comment:
You don't need to compare a `boolean` to `true`, you can just use it as the condition. Technically, `browser.isElementPresent` returns a `Promise<any>`, but that's confusing because it's actually just an alias for `element(locator).isPresent` which returns a `Promise<boolean>`. So really this line could just be:

```typescript
if (await element(by.cssContainingText("th", name)).isPresent()) {
```